cirq/circuits/circuit.py

```diff
@@ -1494,7 +1495,7 @@ def unitary(self,
         result = _apply_unitary_circuit(self, state, qs, dtype)
         return result.reshape((side_len, side_len))
 
-    def final_wavefunction(
+    def final_state_vector(
             self,
             initial_state: 'cirq.STATE_VECTOR_LIKE' = 0,
```

Comment:
Should this be changed into initial_state_vector, or is this too generic?

Reply:
See nomenclature.md. For STATE_VECTOR_LIKE, state_rep or initial_state are ok, since it's not a vector (it can be an int).